config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :merchants, only: [:show], defaults: {format: :json}
```

Comment:
when we have more functionality, maybe consider changing only: to except:? 
But for now, this is great! :)